iocore/net/OCSPStapling.cc

```diff
@@ -185,17 +183,28 @@ ssl_stapling_init_cert(SSL_CTX *ctx, X509 *cert, const char *certname)
   }
 
   if (!cinf->uri) {
-    OCSP_CERTID_free(cinf->cid);
-    cinf->cid = nullptr;
-
     Note("no OCSP responder URI for %s", certname);
-    return false;
+    goto err;
```

Comment:
You can't use std::unique_ptr or ts::PostScript to avoid the forgettable goto?

Reply:
I was following the pattern we use in these SSL files.  We should make a pass and convert all of these to unique_ptr.